aten/src/ATen/native/native_functions.yaml

```diff
@@ -1967,6 +1967,10 @@
     SparseCsrCPU, SparseCsrCUDA: empty_sparse_compressed
     QuantizedCPU, QuantizedCUDA: empty_unknown_quantized
 
+- func: empty.SymInt(SymInt[] size, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None, MemoryFormat? memory_format=None) -> Tensor
```

Comment:
No overload conflict for this one?

Reply:
No, because of https://github.com/pytorch/pytorch/pull/79281

Reply:
Ho right this is done under the hood.